src/main/java/rx/plugins/RxJavaSingleExecutionHookDefault.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package rx.plugins;
+
+/**
+ * Default no-op implementation of {@link RxJavaSingleExecutionHook}
+ */
+/* package */class RxJavaSingleExecutionHookDefault extends RxJavaSingleExecutionHook {
+
+    private static RxJavaSingleExecutionHookDefault INSTANCE = new RxJavaSingleExecutionHookDefault();
```

Comment:
I see similar issues in other places in RxJava codebase Shall I fix it in this PR or create separated one?